packages/components/src/card/styles.ts

```diff
@@ -12,7 +12,7 @@ import { COLORS, CONFIG } from '../utils';
 // (as opposed to the `border` property), the value of the border radius needs
 // to be adjusted by removing 1px (this is because the `box-shadow` renders
 // as an "outer radius").
-const adjustedBorderRadius = `calc(${ CONFIG.cardBorderRadius } - 1px)`;
+const adjustedBorderRadius = `calc(${ CONFIG.radiusLarge } - 1px)`;
```

Comment:
Is this something that we want to keep, or should we switch to `radiusMedium` ? 

Since we're defining a list of radii, I feel like we could try to avoid these exceptions.

Reply:
Yes, sorry if I was unclear. I mean the `border - 1px` calculation, instead of using a `radius` from the list of shared variables.

Reply:
Or it might be a good idea to just change it to `radiusMedium` as both @ciampo and @jasmussen suggested, and ditch the calculation altogether.

Reply:
Without the calculation there is an issue with the full-bleed-content version. Note the tiny white space visible between the border and the red area:

<img width="675" alt="Screenshot 2024-09-04 at 15 48 46" src="https://github.com/user-attachments/assets/820daa6c-90e1-4b84-9659-dbd90c9dbfeb">